README.md

```diff
@@ -1 +1,21 @@
-# nhsuk-tools-chart-components
\ No newline at end of file
+# NHS.UK Tools Chart components
+
+This repository contains the code for NHS.UK Chart components - a port of the [NHS.UK Tools Chart components](https://github.com/NHSDigital/nhsuk-tools-chart-components).
+
+These chart components are designed for visualizing Body Mass Index (BMI), Child Body Mass Index (CBMI), and Blood Pressure (BP) data.
+
+Initially, these charts were integrated into separate codebases for each of the individual tools. To support consistency and reusability across NHS projects, we have migrated these chart components into a dedicated library.
+
+## Installation
+
+Ensure you have Node.js 20.x.x. installed. Usage of NVM is recommended, to allow for easily switching between versions on different projects.
+
+`npm install` will install dependencies.
+
+## Build
+
+`npm run build` will build the project.
+
+## Run
+
+`npm run dev` will run the application locally in development mode.
```

Comment:
is this right?

Reply:
ofcourse not :) i forgot i was in the library . will remove